db/migrate/20220820131321_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Again, the reference name should be `author`. Please refer to the above similar comment for the other migration.

Reply:
Yeah!, :+1: